### PR TITLE
#153114594 Add dashboard feature for failed jobs

### PR DESCRIPTION
**Pull request by geofrocker**
#### What does this PR do?
Add new dashboard tab for unresolved stuff

### Currently
When a User's job goes down, the user is sent a notification email informing him of the same
### Expected
Apart from the email notifications, the user should also have a separate dashboard view of their failed jobs

#### How should this be manually tested.
After cloning the repo, cd into the root of the project:
*Create virtualenv*
`$ virtualenv --python=python3 hc-venv`
`$ source hc-venv/bin/activate`

*run* 
`python manage.py runserver`.

#### What are the relevant pivotal tracker stories?

[#153114593](https://www.pivotaltracker.com/story/show/153114593)
 

#### Relevant links?
https://hc-gnext.herokuapp.com/accounts/dashboard/

#### Screenshot of the dashboard
screen

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,5 +5,7 @@ django_compressor==2.1
 djmail==0.11.0
 futures==3.0.3
 premailer==2.9.6
-psycopg2==2.6.1
+psycopg2==2.7.3
 requests==2.9.1
+gunicorn
```
Comment (TinaMurimi) [inline requirements.txt]: - How is this? A `pip freeze` always gives the package version

Comment (TinaMurimi) [reply requirements.txt]: You are to use `waitress`: https://andela.slack.com/archives/C85EWN51C/p1512985077000006

File: requirements.txt [hunk 2]
```diff
@@ -5,5 +5,7 @@ django_compressor==2.1
...
+whitenoise
```
Comment (TinaMurimi) [inline requirements.txt]: How is this? A `pip freeze` always gives the package version

**Review by TinaMurimi — COMMENTED** [review]
- The application serves static files. Use `whitenoise`. See resources shared and pinned on Slack channel
- You app has an error when trying to login
screen shot 2017-12-11 at 18 33 14

File: requirements.txt (hunk 1 above)
Comment (geofrocker) [reply requirements.txt]: Have changed to waitress, Done!

File: requirements.txt (hunk 2 above)
Comment (geofrocker) [reply requirements.txt]: Noted, have updated, thank you

**Comment by geofrocker** [comment]
We have tested the login with my colleagues and its  working. I was having that error at first but
 fixed

**Comment by geofrocker** [comment]
Iam also using whitenoise

**Comment by geofrocker** [comment]
sc

File: .travis.yml [hunk 3]
```diff
@@ -14,6 +14,7 @@ addons:
 before_script:
   - psql -c 'create database hc;' -U postgres
 script:
+  - pep8 --show-source --show-pep8 hc/
```
Comment (TinaMurimi) [inline .travis.yml]: Line 17 and 19 do the same thing

**Review by TinaMurimi — COMMENTED** [review]
Where are the files used for hosting the app on Heroku?

File: hc/settings.py [hunk 4]
```diff
@@ -77,11 +78,14 @@
 # Default database engine is SQLite. So one can just check out code,
 # install requirements.txt and do manage.py runserver and it works
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME':   './hc.sqlite',
-    }
+    'default': dj_database_url.config()
 }
+# DATABASES = {
```
Comment (TinaMurimi) [inline hc/settings.py]: If this is not being used, delete the code

File: hc/settings.py [hunk 5]
```diff
@@ -111,7 +115,7 @@
 USE_L10N = True
 USE_TZ = True
 
-SITE_ROOT = "http://localhost:8000"
+SITE_ROOT = "https://hc-gnext.herokuapp.com"
```
Comment (TinaMurimi) [inline hc/settings.py]: Use link for group Heroku: https://dev-gnext.herokuapp.com/

Comment (geofrocker) [reply hc/settings.py]: https://hc-gnext.herokuapp.com is the heroku link

File: hc/settings.py (hunk 4 above)
Comment (geofrocker) [reply hc/settings.py]: Have worked on it

File: hc/settings.py (hunk 5 above)
Comment (geofrocker) [reply hc/settings.py]: https://dev-gnext.herokuapp.com is the heroku link

File: templates/accounts/dashboard.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+{% endif %}
+
+{% endblock %}
+
+{% block scripts %}
+{% compress js %}
+<script src="{% static 'js/jquery-2.1.4.min.js' %}"></script>
+<script src="{% static 'js/bootstrap.min.js' %}"></script>
+{% endcompress %}
+{% endblock %}
```
Comment (jonathankamau) [inline templates/accounts/dashboard.html]: A new line is required here. Please fix.

**Review by jonathankamau — CHANGES_REQUESTED** [review]
Does the PT link actually direct to the story? Look into that and fix.

File: templates/accounts/dashboard.html (hunk 6 above)
Comment (geofrocker) [reply templates/accounts/dashboard.html]: Have added a new line. thank you

**Comment by geofrocker** [comment]
Have updated the link to the story.

**Review by jonathankamau — CHANGES_REQUESTED** [review]
Please change the format of the link under `What are the relevant pivotal tracker stories?` to the format I gave in the channel.

**Comment by geofrocker** [comment]
Have worked on the stories link

**Pull request merged**